submissions/selveresta/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,126 @@
+									<p class="popup-text">Files</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/contacts.png" alt="" class="popup-img" />
+									<p class="popup-text">Contacts</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/tanslation.png" alt="" class="popup-img" />
+									<p class="popup-text">Tanslate</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/videos.png" alt="" class="popup-img" />
+									<p class="popup-text">Videos</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/search.png" alt="" class="popup-img" />
+									<p class="popup-text">Search</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/photos.png" alt="" class="popup-img" />
+									<p class="popup-text">Photo</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/apps.png" alt="" class="popup-img" />
+									<p class="popup-text">Apps</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/documents.png" alt="" class="popup-img" />
+									<p class="popup-text">Documents</p>
+								</a>
+							</div>
+							<input type="checkbox" class="more" id="more" />
+							<label for="more" class="label-more">
+								<span class="more-text">More</span>
+							</label>
+							<div class="popup-items">
+								<a href="#" class="popup-item">
+									<img src="./icons/chat.png" alt="" class="popup-img" />
+									<p class="popup-text">Chat</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/classroom.png" alt="" class="popup-img" />
+									<p class="popup-text">Classroom</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/collection.png" alt="" class="popup-img" />
+									<p class="popup-text">Collection</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/mars.png" alt="" class="popup-img" />
+									<p class="popup-text">Mars</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/blogs.png" alt="" class="popup-img" />
+									<p class="popup-text">Blogs</p>
+								</a>
+								<a href="#" class="popup-item">
+									<img src="./icons/promo.png" alt="" class="popup-img" />
+									<p class="popup-text">Promos</p>
+								</a>
+							</div>
+						</div>
+					</li>
+					<li class="nav-item">
+						<input id="box-bell" type="checkbox" class="check-input" />
+						<label for="box-bell" class="label-position">
+							<span class="box-bell"></span>
+						</label>
+					</li>
+					<li class="nav-item">
+						<a href="#" class="profile-link">
+							<img src="./icons/profile.png" alt="profile" class="profile-img" />
+						</a>
+					</li>
+				</ul>
+			</nav>
+		</header>
+		<img src="icons/unnamed.jpg" id="img_main" />
```

Comment:
Read [this article](https://htmlacademy.ru/blog/articles/alt-text) about `alt` attribute and when he must be populated. Sorry for russian but it's a good article.